### PR TITLE
Update v0.3 Default Pins

### PR DESCRIPTION
**Pull request by DetonationEMS**
Add Barometric Pressure sensor and Vehicle Speed Sensor default pins to v0.3x

### CODE REVIEW

File: speeduino/init.ino [hunk 1]
```diff
@@ -1267,6 +1267,7 @@ void setPinMapping(byte boardID)
       pinTrigger3 = 3; //The Cam sensor 2 pin
       pinTPS = A2;//TPS input pin
       pinMAP = A3; //MAP sensor pin
+      pinBaro = A5; //Baro Sensor Input
```
Comment (diy-efi) [inline speeduino/init.ino:1270]: You are adding pinBaro (A5) twice, here (Line 1270) and again later on (Line 1291).

**Review by diy-efi — CHANGES_REQUESTED** [review]
Remove one of the pin definition lines.

**Comment by DeionSi** [comment]
These changes are already in master. Commits d5fc16d and be58be3.

**Comment by DetonationEMS** [comment]
Leaning to pull from VSCode. Didn't mean to re-open this. Sorry.

**Pull request closed without merging**